server/routes/user.js

```diff
@@ -3,21 +3,25 @@ import UserController from '../controllers/UserController';
 import middlewares from '../middlewares';
 
 const {
-  favouriteAuthor, unfavouriteAuthor, favoriteBook, unfavoriteBook
+  favouriteAuthor, unfavouriteAuthor, favoriteBook, unfavoriteBook, editProfile
 } = UserController;
 const {
   verifyToken, authorizeUser, AuthorValidator: { favAuthorValidation },
-  BookValidator: { FavoriteBookValidation }
+  BookValidator: { FavoriteBookValidation }, UserValidator: { updateValidation }
 } = middlewares;
 
 const user = express.Router();
 const FAVOURITES_BASE_URL = '/favourites';
+const UPDATE_BASE_URL = '/update';
```

Comment:
Following REST guidelines, the correct URL for this route should be something like _/user/:userId_ since it is a particular user we are updating.